docs/next/modules/en/pages/troubleshooting/troubleshooting.adoc

```diff
@@ -0,0 +1,220 @@
+
+Understanding the intricacies of both Rancher Turtles and CAPI is crucial for effective troubleshooting. This guide aims to equip you with the knowledge to diagnose and resolve basic problems in Rancher Turtles and CAPI.
+
+== Prerequisites
+
+To effectively troubleshoot Rancher Turtles and CAPI clusters, you need:
+
+* Access to a terminal with kubectl configured to access Rancher management cluster
+* Krew plugin manager for kubectl installed
+* Helm CLI tool installed
+
+You can install Krew by following the instructions at https://krew.sigs.k8s.io/docs/user-guide/setup/install/[Krew installation guide].
+
+== How to check the logs
+
+To chek the logs you can check following pods in namespaces:
+
+
+|===
+| Namespace | Pod | Description
+
+| rancher-turtles-system
+| rancher-turtles-cluster-api-operator
+| Manages the installation and lifecycle of Cluster API providers in the management cluster
+
+| rancher-turtles-system
+| rancher-turtles-controller-manager
+| Handles Rancher integration and manages the custom resources for Turtles
+
+| rancher-turtles-system
+| caapf-controller-manager
+| Controls the Cluster API Provider Framework for custom infrastructure providers
+
+| capi-system
+| capi-controller-manager
+| Core Cluster API controller that reconciles main cluster resources
+
+| rke2-bootstrap-system
+| rke2-bootstrap-controller-manager
+| Manages the bootstrap configuration for RKE2 nodes
+
+| rke2-control-plane-system
+| rke2-control-plane-controller-manager
+| Handles the initialization and management of RKE2 control plane components
+
+| *-system
+| *-controller-manager
+| Handles the initialization and management of Infrastructure provider components
+
+| $-bootstrap-system
+| $-bootstrap-controller-manager
+| Manages the bootstrap configuration for custom providers
+
+| $-control-plane-system
+| $-control-plane-controller-manager
+| Handles the initialization and management of custom providers control plane components
+|===
+
+'*' - depending on the infrastructure provider in use
+'$' - depending on controlplane/bootstrap provider in use, e.g. kubeadm or rke2
+
+
+The best method to monitor logs from different pods is by using the following method.
+
+[source,bash]
+----
+kubectl krew install stern
+----
+
+Then you can tail logs from all pods in real time.
+
+[source,bash]
+----
+kubectl stern . -n rancher-turtles-system -n rke2-bootstrap-system -n rke2-control-plane-system -n capi-system
+----
+
+== How to manage Rancher Turtles and CAPI resources
+
+*List Rancher and Cluster API resources*
+
+Often users are asking how to deal with long list of different CRDs created by CAPI and Rancher, here is some idea how you can deal with it.
+
+. List all CAPI and Rancher related credentials
++
+[source,bash]
+----
+kubectl api-resources | grep 'cattle.io\|cluster.x-k8s.io'
```

Comment:
`kubectl get cluster-api`  is also helpful in listing all the CAPI related resources.

Reply:
ok, I will change it